lib/licenses.nix

```diff
@@ -118,6 +118,11 @@ in mkLicense lset) ({
     fullName = "Apache License 2.0 with LLVM Exceptions";
   };
 
+  arachnosoft = {
+    fullName = "Arachnosoft License";
+    url = "http://www.arachnosoft.com/main/download.php?id=soundfont-sf2"; # license included in sources
+  };
```

Comment:
I think people (and programs) will expect the license URL to be a place where the text of the license can be seen, not a software download, so if this license is not online at all, it probably makes more sense to just use `licenses.free` or `licenses.unfree`.